README.md

```diff
@@ -48,32 +38,30 @@ Then, the SDK can be accessed through `window.aptosSDK`.
 
 ## Usage
 
-Initialize `Aptos` to access the SDK API.
+Create an `Aptos` client in order to access the SDK's functionality.
 
 ```ts
-// initiate the main entry point into Aptos SDK
-const aptos = new Aptos();
-```
-
-If you want to pass in a custom config
+import {{ Aptos, AptosConfig, Network }} from "@aptos-labs/ts-sdk"
 
-```ts
-// an optional config information for the SDK client instance.
-const config = new AptosConfig({ network: Network.LOCAL }); // default network is devnet
+// You can use AptosConfig to choose which network to connect to
+const config = new AptosConfig({{ network: Network.TESTNET }});
```

Comment:
Why testnet and not devnet?

Reply:
That was the advice I got for which one was better for the developer experience. Devnet resets, which seems like less production ready, so if both are testing environments, TESTNET seems more recommended.